docs/running-on-yarn.md

```diff
@@ -495,6 +495,15 @@ To use a custom metrics.properties for the application master and executors, upd
   name matches both the include and the exclude pattern, this file will be excluded eventually.
   </td>
 </tr>
+<tr>
+  <td><code>spark.hadoop.callerContext</code></td>
```

Comment:
doesn't match above actual config spark.upstreamApp.callerContext.  how about:  spark.log.callerContext

If I'm running spark in standalonde mode with master/worker and reading from hdfs, the caller context would still work on the hdfs side, right?  So this isn't just a  spark on yarn config and should move to general configuration section but mention applies to yarn/hdfs.